parallel-crypto3/libs/parallel-zk/include/nil/crypto3/zk/snark/systems/plonk/placeholder/permutation_argument.hpp

```diff
@@ -110,8 +113,16 @@ namespace nil {
                         math::polynomial_dfs<typename FieldType::value_type> V_P(basic_domain->size() - 1,
                                                                                  basic_domain->size());
 
-                        std::vector<math::polynomial_dfs<typename FieldType::value_type>> g_v = S_id;
-                        std::vector<math::polynomial_dfs<typename FieldType::value_type>> h_v = S_sigma;
+                        std::vector<math::polynomial_dfs<typename FieldType::value_type>>

```

Comment:
Maybe we still can do something like std::vector<math::polynomial_dfs<typename FieldType::value_type>> g_v (S_id.begin(), S_id.end()); ???

Reply:
Done.